src/ResourceManager/KeyVault/Commands.KeyVault.Test/ScenarioTests/KeyVaultManagementController.cs

```diff
@@ -38,10 +40,11 @@ namespace Microsoft.Azure.Commands.KeyVault.Test
     public class KeyVaultManagementController
     {
 
-        private CSMTestEnvironmentFactory csmTestFactory;
+        private LegacyTest.CSMTestEnvironmentFactory csmTestFactory;
```

Comment:
I don't think the new way of testing is fully available. The csmTestFactory is still the only option for a bunch of functionality.